ideas/293-ulc-integration/README.md

```diff
@@ -0,0 +1,101 @@
+---
+id: 293-ulc-integration
+title: ULC integration
+status: draft
+created: 2018-08-06
+category: core
+lead-contributor: b00ris
+contributors:
+    - b00ris Go dev
+    - jeka Go dev
+exit-criteria: no
+success-metrics: no
+clear-roles: no
+future-iteration: no
+roles-needed:
+    - Go dev
+    - Go dev
+    - Clojure dev
+    - QA
+okrs:
+    - Core. [P0] LES and/or ULC are operational and used by at least 10% of all users.
+    - Research. Integrate and test ULC in a manner “graceful downgrade”: use infura by default and if it fails, start ULC node.
+---
+
+## Preamble
+
+    Idea: 293-ulc-integration
+    Title: ULC integration
+    Status: Draft
+    Created: 2018-08-06
+    Requires (*optional): #254
+
+## Summary
+Status app can use LES and ULC as Ethereum providers instead of Infura.
+
+## Swarm Participants
+- Lead Contributor: @b00ris
+- Contributor: @jeka
+- Contributor: @mandrigin
+- Contributor: TBD
+- QA: TBD
+
+## Product Overview
+We need LES client enabled on mobile device to be really decentralized and to enable all web3 features for dApps.
+
+### Requirements & Dependencies
+Depends on research swarm #254
```

Comment:
LES integration doesn't depend on this, it's bringing back previous work and fixing bugs around it.

Reply:
Yes, you are right.  It blocks ULC integration. I've fixed it.